apps/src/templates/instructions/HintPrompt.jsx

```diff
@@ -26,7 +26,12 @@ const HintPrompt = ({
       textToSpeechEnabled={textToSpeechEnabled}
     >
       <p>{message}</p>
-      <LegacyButton type="cancel" onClick={onConfirm} style={{marginRight: 5}}>
+      <LegacyButton
```

Comment:
Sorry, one other thought -- should this have an `aria-labelledby`? Wondering what the experience is like for a screen reader here, where I think it would just read "Yes" to you, rather than what you're saying yes to. I think you could use the `p` tag above if you gave it an ID?

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby

(to be honest, only skimmed these docs, so totally might not be the right approach!)

Reply:
oic, the label you include for the lightbulb kind of handles this. Feels non-standard, but probably does the trick!

Reply:
(on the other hand, if you didn't click yes immediately, then navigated back to the "Yes" button, you'd have no context, I think. So maybe worth considering?)

Reply:
This is so cool! Added the label and then checked with the screenreader. Great suggestion!